rust/src/plugin.rs

```diff
@@ -6,6 +6,17 @@ pub struct Plugin<'a> {
 }
 
 impl<'a> Plugin<'a> {
+    pub fn from(id: i32, context: &'a Context) -> Plugin<'a> {
+        Plugin {
+            id,
+            context: context,
+        }
+    }
+
+    pub fn as_i32(&self) -> i32 {
+        self.id
+    }
```

Comment:
I needed to add ways to go to and from the primitive values in the rust code in `extism_nif/src/lib.rs`

Reply:
I think these can go away if i change to pointer as zach suggested.